docker/reqmgr2ms-unmerged/README.md

```diff
@@ -0,0 +1,31 @@
+
+
+### build image
+docker build -t cmssw/reqmgr .
```

Comment:
If you copy-and-paste instructions then you should reflect them to the service you apply them to, i.e. in instruction document it uses `cmssw/reqmgr` but here your service is `cmssw/reqmgr2ms-unmerged`. Please adjust accordingly to avoid wrong instructions.

Reply:
ok. fixing this one too